### ISSUE TITLE
Runtime logging information

### ISSUE THREAD

**Issue by zeinramadan** [AndriyMulyar/bert_document_classification#21]
Does it take a long time before logging information starts being displayed in the terminal? I started running the training script, it displays information about loading the model weights etc but then it is stagnant after this line: 

`
Weights from pretrained model not used in DocumentBertLSTM: ['cls.predictions.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.decoder.weight', 'cls.seq_relationship.weight', 'cls.seq_relationship.bias']
`

Is this normal? does the architecture first encode the entire data before training and displaying information related to epochs, training loss, validation loss, etc.. ?
